### PR TITLE
Fixed Format/Source Issue in Malaria Case Study

### PR DESCRIPTION
**Pull request by JinjinJanelle**
Added source for original version and fixed the format.

### CODE REVIEW

File: 03-casestudies.Rmd [hunk 1]
```diff
@@ -352,14 +352,17 @@ Interactive graph have its own specific advantages. It helps bridge the gap betw
 ### Malaria
 The authors of Vizwiz redesigned "The Seasonality of Confirmed Malaria Cases in Zambia Southern Province" by pointing out what works well, what could be improved, and why their new visualization will be better [@vizwiz].
 
-This chart below shows number of malaria cases reported for health facilities and community health workers and a comparison between the two over the years. From this chart we can clearly see that as summer approaches, cases of malaria increase indicating a seasonality. The colors are also distinct from each other. 
 Original Version: ![Original visualization of malaria cases](images/malaria_original.png)
+([Source]https://www.path.org/visualize-no-malaria/)
```
Comment (syragrg) [inline 03-casestudies.Rmd]: should the link be within another parenthesis?

Comment (JinjinJanelle) [reply 03-casestudies.Rmd]: I changed it. Can you see?

Comment (JinjinJanelle) [reply 03-casestudies.Rmd]: I fixed again!

**Review by syragrg — APPROVED** [review]
looks good! thanks for the changes!

**Review by jacksonh2 — APPROVED** [review]
Looks good! The format looks a lot cleaner now. Great work! I will go ahead and merge it

**Pull request merged**